IviDriverCore/1.0/Spec/IviDriverCore.md

```diff
@@ -0,0 +1,770 @@
+
+Drivers that otherwise comply with the *IVI Driver Core Specification* may use the format described in *IVI-3.1 Architecture Specification*.  That format shall be an acceptable alternate format for drivers that conform to that specification.
+
+### README.md Documentation
+
+The driver README.md file includes details that will help users get started with the driver.  It shall contain:
+
+- The location of the `___Introduction to <DriverIdentifier>___.<DocumentExtension>` documentation file.
+- The location of the help documentation file(s).
+- The text of the compliance documentation. The compliance documentation includes items normally associated with a `README.md` file, such as driver version information, known issues, and driver dependencies. The `README.md` file may also contain the location of a separate compliance document that also contains the compliance documentation.
+
+The `README.md` file may contain additional information that users may need to know, or find useful, before using the driver.
+
+## File Versioning
+
+IVI driver Windows DLLs shall contain a Windows version resource with the three entries: `CompanyName`, `ProductName`, and `FileVersion`.
+
+- `CompanyName` is a string that contains the value of the Driver Vendor or Driver Vendor property that the IVI driver returns.
+
+- `ProductName` is a string that includes the value of the Driver Identifier property that the IVI driver returns.
+
+- `FileVersion` is a string in the following format:
+  
+  `MajorVersion.MinorVersion.BuildVersion[.InternalVersion]`
+
+`MajorVersion`, `MinorVersion`, `BuildVersion`, and `InternalVersion` are decimal numbers greater than or equal to zero and less than or equal to 65535. The MajorVersion shall not be zero, except for a pre-release version of the initial release of a driver. Each number may contain leading zeros but may not exceed 5 digits. The number fields are separated by periods, with no embedded spaces. The maximum valid FileVersion is 65535.65535.65535.65535.
+
+A FileVersion string shall contain at least a `MajorVersion`, `MinorVersion`, and `BuildVersion`. A driver shall be released with an incremented `MajorVersion` or `MinorVersion` number when any of the following conditions are true:
+
+- The new version of the driver contains changes in its API syntax.
+
+- The new version of the driver contains significant changes to its semantics.
+
+A driver shall be released with at least an incremented `BuildVersion` number when any of the following conditions are true:
+
+- A new bitness of the driver is provided.
+
+- The driver is modified in a way that does not require a `MajorVersion` or `MinorVersion` update.
+
+If a driver vendor provides both a 32-bit and a 64-bit version of a driver, the 32-bit and 64-bit DLLs shall have the same `MajorVersion`, `MinorVersion`, and `BuildVersion`. Notice that when a 64-bit version of the driver is initially made available, the 32-bit version must be updated at the same time.
```

Comment:
It's good we call out 32 and 64-bit releases must be simultaneous and use matching versions. What about when a driver supports multiple OSes? Is this being omitted intentionally?